invenio_requests/assets/semantic-ui/js/invenio_requests/timeline/TimelineFeed.js

```diff
@@ -31,6 +31,12 @@ class TimelineFeed extends Component {
     getTimelineWithRefresh();
   }
 
+  componentDidUpdate() {
+    if (window.MathJax) {
+      window.MathJax.typeset();
+    }
```

Comment:
Can we trigger it only when the `timeline` prop has changed and contains more than 0 results, and not for every cmp update?
```suggestion
    window.MathJax?.typeset();  // trigger MathJax magic
```